terraform/gitlab/gitlab.tf.json.template.py

```diff
@@ -1936,20 +1936,20 @@ def merge(sets: Iterable[Iterable[str]]) -> Iterable[str]:
                                     )
                                     for command in [
                                         sq(
+                                            # docker volume prune --force --all
                                             'docker',  # The docker …
-                                            'image',  # … image command …
+                                            'volume',  # … volume command …
                                             'prune',  # … to delete, …
                                             '--force',  # … without prompting for confirmation, …
-                                            '--all',  # … all images …
-                                            f'--filter "until={90 * 24}h"',  # … except those from more recent builds …
+                                            '--all',  # … all unused volumes …
                                             '> /proc/1/fd/1',  # … with output sent to the container's STDOUT.
                                         ),
                                         sq(
                                             'docker',  # The docker …
-                                            'buildx',  # … buildx command …
+                                            'system',  # … system command …
                                             'prune',  # … to delete, …
                                             '--force',  # … without prompting for confirmation, …
-                                            '--all',  # … all images …
+                                            '--all',  # … all unused images …
```

Comment:
Could this change be moved to the previous commit? Or a new, separate commit? It seems unrelated to the semantic changes.

Reply:
It is related to the other changes in this commit as now the option `--all` is now being called on the `docker system prune` command instead of `docker buildx prune`.

The `--all` option in `docker buildx prune` is described:
```
$ docker buildx prune --help
Remove build cache
Options:
  -a, --all                    Include internal/frontend images
```

compared to `docker system prune`:
```
$ dockeer system prune --help
Remove unused data
Options:
  -a, --all             Remove all unused images not just dangling ones
```